### PR TITLE
Feature/mc 11639 create subnet

### PR DESCRIPTION
**Pull request by AnKhangC**
#### Changes made
Added documentation for list and create subnetwork.

#### Related PRs
- [MC-11639] (https://cloud-ops.atlassian.net/browse/MC-11639)
- [MC-11641] (https://cloud-ops.atlassian.net/browse/MC-11641)

[MC-11639]: https://cloudops.atlassian.net/browse/MC-11639
[MC-11641]: https://cloudops.atlassian.net/browse/MC-11641

### CODE REVIEW

**Review by fyrnaga — CHANGES_REQUESTED** [review]
Adding the _subnetworks.md is not sufficiant. You need to modify the index.html.md as well to include the page in the right place/section/order.

Also, make sure the attributes are not valid keys for production. It is also offuscate environments and id to something meaningful for public audience

File: source/includes/gcp/_subnetworks.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+`fingerprint`<br/>*string* | A base64-encoded string. A hash of the label's contents and used for optimistic locking
+`gatewayAddress`<br/>*string* | Second address in the primary IP range for the subnet
+`ipCidrRange`<br/>*string* | Primary IP address range for the following resources: VM instances, internal load balancers, and internal protocol forwarding
+`kind`<br/>*string* | Type of the resource
+`network`<br/>*string* | Server-defined URL for the VPC network that contains the subnet
+`privateIpGoogleAccess`<br/>*string* | Whether the Private Google Access is configured
+`region`<br/>*Array[Disk]* | Server-defined URL for the region name
+`selfLink`<br/>*string* | Server-defined URL for this resource
+`shortNetwork`<br/>*string* | Display name of the VPC network that contains the subnet
+`id`<br/>*UUID* | The id of the instance
```
Comment (GuillaumeVialle) [inline source/includes/gcp/_subnetworks.md]: The id of the `subnet`

File: source/includes/gcp/_subnetworks.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+### Subnetworks
```
Comment (GuillaumeVialle) [inline source/includes/gcp/_subnetworks.md]: Subnets

File: source/includes/gcp/_subnetworks.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+Create and manage your subnetworks. Subnetworks belongs to a network.
```
Comment (GuillaumeVialle) [inline source/includes/gcp/_subnetworks.md]: subnetworks -> subnets

File: source/includes/gcp/_subnetworks.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+<!-------------------- LIST SUBNETWORKS -------------------->
+
+#### List subnetworks
```
Comment (GuillaumeVialle) [inline source/includes/gcp/_subnetworks.md]: Same here

File: source/includes/gcp/_subnetworks.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+`region`<br/>*Array[Disk]* | Server-defined URL for the region name
+`selfLink`<br/>*string* | Server-defined URL for this resource
+`shortNetwork`<br/>*string* | Display name of the VPC network that contains the subnet
+`id`<br/>*UUID* | The id of the instance
+`name`<br/>*string* | The display name of the subnetwork.
+`shortRegion`<br/>*string* | A short version of the region name
+
+<!-------------------- CREATE A SUBNETWORK -------------------->
+
+#### Create a subnetwork
```
Comment (GuillaumeVialle) [inline source/includes/gcp/_subnetworks.md]: same here

File: source/includes/gcp/_subnetworks.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/subnetworks</code>
+
+Create a new subnetwork
+
+Required | &nbsp;
+------- | -----------
+`name`<br/>*string* | The display name of the instance
```
Comment (GuillaumeVialle) [inline source/includes/gcp/_subnetworks.md]: instance -> subnet

File: source/includes/gcp/_subnetworks.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+  "name": "my-subnet",
+  "shortRegion": "northamerica-northeast1",
+  "ipCidrRange": "10.0.0.0/9",
+  "network": "https://www.googleapis.com/compute/v1/projects/my-project/global/networks/my-network"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/subnetworks</code>
+
+Create a new subnetwork
```
Comment (GuillaumeVialle) [inline source/includes/gcp/_subnetworks.md]: subnet

File: source/includes/gcp/_subnetworks.md (hunk 1 above)
Comment (AnKhangC) [reply source/includes/gcp/_subnetworks.md]: 0ef6555

File: source/includes/gcp/_subnetworks.md (hunk 2 above)
Comment (AnKhangC) [reply source/includes/gcp/_subnetworks.md]: 0ef6555

File: source/includes/gcp/_subnetworks.md (hunk 3 above)
Comment (AnKhangC) [reply source/includes/gcp/_subnetworks.md]: 0ef6555

File: source/includes/gcp/_subnetworks.md (hunk 4 above)
Comment (AnKhangC) [reply source/includes/gcp/_subnetworks.md]: 0ef6555

File: source/includes/gcp/_subnetworks.md (hunk 5 above)
Comment (AnKhangC) [reply source/includes/gcp/_subnetworks.md]: 0ef6555

File: source/includes/gcp/_subnetworks.md (hunk 6 above)
Comment (AnKhangC) [reply source/includes/gcp/_subnetworks.md]: 0ef6555

File: source/includes/gcp/_subnetworks.md (hunk 7 above)
Comment (AnKhangC) [reply source/includes/gcp/_subnetworks.md]: 0ef6555

**Comment by fyrnaga** [comment]
should we renamed the endpoint as well?

**Comment by AnKhangC** [comment]
> should we renamed the endpoint as well?

@fyrnaga Do you mean the name of the file?

**Review by GuillaumeVialle — APPROVED** [review]
LGTM

**Comment by fyrnaga** [comment]
> > should we renamed the endpoint as well?
> 
> @fyrnaga Do you mean the name of the file?

that would be the file, the uri and the code to point to subnet instead of subnetworks.... I am saying we must, but it would be a good idea to ask.

**Review by fyrnaga — APPROVED** [review]
LGTM

**Pull request merged**